src/components/AccessKeysColumns/columns.tsx

```diff
@@ -0,0 +1,99 @@
+import { AccessKey } from '@bsv/spv-wallet-js-client';
+import { Link } from '@tanstack/react-router';
+import { ColumnDef } from '@tanstack/react-table';
+
+import { ArrowUpDown } from 'lucide-react';
+
+import { Badge } from '@/components/ui/badge.tsx';
+
+import { Button } from '@/components/ui/button.tsx';
```

Comment:
To increase readability while have many imports

Reply:
It is a thing which could be good to discuss in our team:
- because we have more than one javascript project
- we should try to keep consistency between our projects (concerning code style etc.)
- we have several developers with js/ts/react experience - maybe they would have some points and suggestions about that